tfs/collection.py

```diff
@@ -66,40 +66,40 @@ class TfsCollection(metaclass=_MetaTfsCollection):
 
     .. code-block:: python
 
-        >>> # All TFS attributes must be marked with the Tfs(...) class,
-        ... # and generated attribute names will be appended with _x / _y
-        ... # depending on files found in "./example"
-        ... class ExampleCollection(TfsCollection):
-        ...     beta = Tfs("beta_phase_{}.tfs")  # A TFS attribute
-        ...     other_value = 7  # A traditional attribute.
+        # All TFS attributes must be marked with the Tfs(...) class,
```

Comment:
**MAINTENANCE**

All of this is to get rid of warnings when generating the documentation.